Library/OptimizelyExperiment.swift

```diff
@@ -30,4 +30,15 @@ extension OptimizelyExperiment {
         userAttributes: optimizelyUserAttributes(with: project, refTag: refTag)
       ) ?? .control
   }
+
+  // Returns variation via getVariation for native_project_cards experiment
+  static func nativeProjectCardsExperimentVariant() -> OptimizelyExperiment.Variant {
```

Comment:
I think it might be preferable to keep the function that activates the experiment private to the VM that uses it, unless we intend to activate from multiple places in the app (we don't, at this time).